dnf-behave-tests/features/alias-command.feature

```diff
@@ -228,3 +228,16 @@ Scenario: One disabled config does not affect others
         | Action        | Package                               |
         | install       | dnf-ci-package-0:1.0-1.x86_64         |
   Given I successfully execute dnf with args "remove dnf-ci-package"
+
+
+@bz1680482
+Scenario: Backslash ends the recursive processing and the '\' is stripped
+   When I execute dnf with args "alias add install='\install'"
+   Then the exit code is 0
```

Comment:
You could use:
```gherkin
   Given I successfully execute dnf with args "alias add install='\install'"
```
Instead of the two lines above.

Reply:
Fixed.